tests/regression/whole_axis_access.cpp

```diff
@@ -45,7 +46,7 @@ namespace {
             stencil_backend_t(),
             TypeParam::make_grid(),
             out,
-            sid::rename_dimension<dim::k, integral_constant<int_t, 3>>(TypeParam::make_storage(in)),
+            sid::rename_dimensions(TypeParam::make_storage(in), 2_c, 3_c),
```

Comment:
```suggestion
            sid::rename_dimensions(TypeParam::make_storage(in), dim::k(), 3_c),
```
Looks slightly more readable to me. On the other hand, you have to know that `dim::k()` is the same as `2_c` if you write such code, so feel free to keep it if you prefer…